src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3377,7 +3377,7 @@ private string BuildXBindEvalFunction(XamlMemberDefinition member, XamlObjectDef
 				.Where(p => !p.StartsWith("global::"))  // Don't include paths that start with global:: (e.g. Enums)
 				.Select(p => $"\"{p}\"");
 
-			var pathsArray = propertyPaths.properties.Any()
+			var pathsArray = formattedPaths.Any()
```

Comment:
I think I fixed an issue here? It was generating a `new [] {}` for the propertyPaths since I was `x:Bind`ing to a `const` so it had `global::` in the path. Wouldn't compile since it couldn't infer the type of the array

Reply:
Indeed! Nice catch!